cibuildwheel/__main__.py

```diff
@@ -368,7 +372,8 @@ def detect_warnings(platform: str, build_options: BuildOptions) -> List[str]:
         option_value = getattr(build_options, option_name)
 
         if option_value and ('{python}' in option_value or '{pip}' in option_value):
-            warnings.append(option_name + ": '{python}' and '{pip}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead.")
+            msg = ": '{python}' and '{pip}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead."
+            warnings.append(option_name + msg)
```

Comment:
I wonder if this would be better as f-string?

```suggestion
            # p.s. in an f-string, double braces means literal single brace
            warnings.append(f"{option_name}: '{{python}}' and '{{pip}}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead.")
```